### ISSUE TITLE
how to reduce iterations num which is needed to reach the convergence?

### ISSUE THREAD

**Issue by MingPap** [AlexeyAB/darknet#2090]
@AlexeyAB  Hi~

If I reduce the ***width*** and ***height*** parameters, iterations num which is needed to reach the convergence will reduce as well ?

**Comment by MingPap** [comment]
@AlexeyAB 

What about increasing the learning_rate?

**Comment by MingPap** [comment]
I think maybe I ask too much :)

**Comment by MingPap** [comment]
@AlexeyAB 

One more question.

if the loss fluctuate at 0.7 in the end, will it be normal?

**Comment by MingPap** [comment]
@AlexeyAB

I'm so sorry. one mor question again

if I use Adam, will convergence time reduce?

**Comment by AlexeyAB** [comment]
@MingPap Hi,

> if I use Adam, will convergence time reduce?

No. There is already used very good optimizer with momentum and decay: https://github.com/AlexeyAB/darknet/issues/1845#issuecomment-440283505


> What about increasing the learning_rate?

Yes, you can.


> if the loss fluctuate at 0.7 in the end, will it be normal?

It depends on model and dataset. For the most cases loss < 1.0 is normal.
It is much better higher mAP than lower loss: https://github.com/AlexeyAB/darknet#when-should-i-stop-training

Try to train by using this command by using the latest version from this repository:
`./darknet detector train data/obj.data yolo-obj.cfg darknet53.conv.74 -map`

When mAP is no longer increasing, then you can stop the training.

**Comment by MingPap** [comment]
@AlexeyAB

I use Ubuntu server to train on line, so I can’t use ***map*** command to see mAP graph directly when training.

Thank you for help!

**Comment by MingPap** [comment]
@AlexeyAB

To get ***yolo_obj_final.weight***, the iteration num is a unchanged number, or the num will change automatically depending on the loss?
